test/test_browser.py

```diff
@@ -1895,8 +1895,12 @@ def test_gles2_emulation(self, args):
   def test_clientside_vertex_arrays_es3(self):
     self.btest('clientside_vertex_arrays_es3.c', reference='gl_triangle.png', args=['-sFULL_ES3', '-sUSE_GLFW=3', '-lglfw', '-lGLESv2'])
 
-  def test_emscripten_api(self):
-    self.btest_exit('emscripten_api_browser.c', args=['-sEXPORTED_FUNCTIONS=_main,_third', '-lSDL'])
+  @parameterized({
+    'base': (['-sEXPORTED_FUNCTIONS=_main,_third', '-lSDL'],),
+    'closure_strict': (['-sEXPORTED_FUNCTIONS=_main,_third', '-lSDL', '--closure=1', '-sSTRICT'],),
+  })
+  def test_emscripten_api(self, args):
```

Comment:
Can you confirm the `closure_strict` version fails without your change?

Reply:
I was hoping to see that on the CI job for my other PR (https://github.com/emscripten-core/emscripten/pull/20739) but so far I don't see that it failed. I'm waiting for the CI job to finish.